lib/subscriber.js

```diff
@@ -0,0 +1,48 @@
+'use strict';
+
+var _ = require('underscore')
+  , service = require('./service').service;
+
+module.exports = {
+  find: function(phoneNumber) {
+    return service().bindings.list()
+      .then(function(bindings) {
+        return _.map(bindings, function(binding) {
+          return {
+            id: binding.sid,
+            phoneNumber: binding.address,
+            movies: binding.tags
+          };
+        });
+      })
+      .then(function(subscribers) {
+        return _.first(_.filter(subscribers, function(subscriber) {
```

Comment:
I'm counting on it, if that is the case, it will return `undefined`. The other option was to return anempty array and then check the result using something like `_.isEmpty(result)`.

The advantage with the current approach in my opinion is that `undefined` in an if clause is evaluated as `false`. That's why I can use this:

```javascript
if (subscriber) { // subscriber can be an object or be undefined.
  ...
```

What do you think?